website/docs/docs/cloud/dbt-cloud-ide/git-commit-signing.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Git commit signing"
+description: "Learn how to sign your Git commits when using the IDE for development."
+sidebar_label: Git commit signing
+---
+
+# Git commit signing <Lifecycle status="Enterprise" />
+
+To improve identity protection/impersonation and enhance security, you can sign your Git commits when pushing them to the repository. Using your signature, a Git provider can cryptographically verify it and mark the commit as "verified", giving people increased confidence about the origin of the commit. 
+
+You can configure dbt Cloud to sign your Git commits when using the IDE for development. To set up, enable the feature in dbt Cloud, generate a GPG keypair, and upload the public key to your Git provider to use for signature verification.  
+
+
+## Prerequisites 
+
+- GitHub or GitLab is your Git provider. Currently, Azure DevOps is not supported.
```

Comment:
do we want to say azure is coming soon? or is this not the case

Reply:
@GideonShils @greg-mckeon

Reply:
it's not possible on Azure, so it won't be supported.